[16.0][IMP] shipment_advice: Add action on shipment advice to print delivery slip

tuantrantg opened the pull request.

sbejaoui committed:
[REF] shipment_advice: refactor action_done

sbejaoui committed:
[IMP] add company config to run shipment advice in queue job

sbejaoui committed:
[IMP] run shipment advice validation in queue jobs

sbejaoui committed:
[IMP] shipment_advice async: manage picking validation error

sbejaoui committed:
[IMP] check that undone move are unplaned after validation

sbejaoui committed:
[IMP] picking can be planned if shipment advice in error

sbejaoui committed:
[FIX] fix queue channel name

sbejaoui committed:
[IMP] allow load/unlaod to sa in error state

sbejaoui committed:
[IMP] check all picking done before setting the sa to done

sbejaoui committed:
[IMP] add job description

tuantrantg committed:
[IMP] shipment_advice: Add action on shipment advice to print delivery slip

nilshamerlinck commented on shipment_advice/views/shipment_advice.xml:
```diff
@@ -357,4 +357,20 @@
+        <field
+            name="code"
+        >picking_type = env.ref("stock.picking_type_out")
+pickings = records.mapped('loaded_picking_ids').filtered(lambda p: p.picking_type_id == picking_type and p.state != 'cancel')
```
better creating a method for that to allow for easy overriding if needed

tuantrantg replied:
hi @nilshamerlinck the PR has been updated

tuantrantg committed:
[ADD] shipment_advice_cash_on_delivery: Add action on shipment advice to print cash on delivery invoices

giarve committed:
Added translation using Weblate (Spanish)

giarve committed:
Added translation using Weblate (Spanish)

giarve committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (22 of 22 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice_planner
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice_planner/es/

giarve committed:
Added translation using Weblate (Spanish)

giarve committed:
Translated using Weblate (Spanish)

Currently translated at 83.3% (15 of 18 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-stock_dock
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-stock_dock/es/

joelkm committed:
Translated using Weblate (Spanish)

Currently translated at 11.1% (24 of 215 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (18 of 18 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-stock_dock
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-stock_dock/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 11.1% (24 of 215 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 22.7% (49 of 215 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice/es/

joelkm committed:
Translated using Weblate (Spanish)

Currently translated at 22.7% (49 of 215 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice/es/

joelkm committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (215 of 215 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (215 of 215 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice/es/

Commit:
Added translation using Weblate (Spanish)

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 39.4% (43 of 109 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice_planner_toursolver
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice_planner_toursolver/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 95.4% (104 of 109 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice_planner_toursolver
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice_planner_toursolver/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice_planner_toursolver_queue_job
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice_planner_toursolver_queue_job/es/

OCA-git-bot committed:
Merge PR #114 into 16.0

Signed-off-by jbaudoux

Commit:
[UPD] Update shipment_advice.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-transport-16.0/stock-logistics-transport-16.0-shipment_advice
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-transport-16-0/stock-logistics-transport-16-0-shipment_advice/
